tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -1063,112 +1082,118 @@ def test_continuous_reads_during_pod_restart(self):
+            master_node_obj=self.node_master_list[0], health_obj=self.hlth_master_list[0],
+            down_method=const.RESTORE_DEPLOYMENT_K8S)
+        # Assert if empty dictionary
+        assert_utils.assert_true(resp[1], "Failed to shutdown/delete pod")
+        pod_name = list(resp[1].keys())[0]
+        self.deployment_name = resp[1][pod_name]['deployment_name']
+        self.deployment_backup = resp[1][pod_name]['deployment_backup']
+        self.restore_method = resp[1][pod_name]['method']
+        assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+        LOGGER.info("Step 2: Successfully shutdown data pod %s. Verified cluster and "
+                    "services states are as expected & remaining pods status is online.", pod_name)
 
-        remain_pod_list = list(filter(lambda x: x != pod_name, pod_list))
-        LOGGER.info("Step 4: Verify services status on remaining pods %s are in online state",
-                    remain_pod_list)
+        LOGGER.info("Step 3: Perform READ-Verify on data written in Step 1")
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix,
+                                                    skipwrite=True, skipcleanup=True,
+                                                    nclients=10, nsamples=10)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 3: Performed WRITEs/READs-Verify with variable sizes objects.")
 
-        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=remain_pod_list, fail=False)
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 4: Services of pod are in online state")
+        LOGGER.info("Step 4: Perform WRITEs/READs-Verify with variable object sizes.")
+        if CMN_CFG["dtm0_disabled"]:
+            self.test_prefix = 'test-34073-1'
 
-        LOGGER.info("Step 5: Perform WRITEs with variable object sizes. (0B - 512MB(VM)/5GB(HW))")
-        users = self.mgnt_ops.create_account_users(nusers=1)
-        self.test_prefix = 'test-34073'
-        self.s3_clean.update(users)
         resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
                                                     log_prefix=self.test_prefix, skipread=True,
                                                     skipcleanup=True, nclients=10, nsamples=10)
         assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 5: Performed WRITEs with variable sizes objects.")
+        LOGGER.info("Step 4: Performed WRITEs/READs-Verify with variable object sizes.")
 
-        LOGGER.info("Step 6: Perform READs and verify DI on the written data in background")
+        LOGGER.info("Step 5: Perform READs and verify DI on the data in background")
+        self.test_prefix = 'test-34073'
```

Comment:
do we need this line, if dtm true then only test_prefix will be updated isn't it?

Reply:
It is needed. Because this will be used in both dtm enabled and disabled scenarios.